CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+## [1.0.2]
```

Comment:
1.1.0, since it's a new feature and not a bugfix (don't forget to update that everywhere)